src/common/lib/python/common_extension.cc

```diff
@@ -173,6 +173,11 @@ static PyObject *PyObjectID_richcompare(PyObjectID *self,
   return result;
 }
 
+static PyObject *PyObjectID_redis_shard_hash(PyObjectID *self) {
```

Comment:
Can we document the fact that this hash function must be the same as the one used in `get_redis_context` in `redis.cc`?